os/include/tinyara/log_dump/log_dump.h

```diff
@@ -24,6 +24,9 @@
  * Definitions
  ********************************************************************************/
 
+ #define LOGDUMP_SAVE_START	"1"
```

Comment:
Is there any reason to define integer as string?
How about changing it to string like `#define LOGDUMP_SAVE_START "logdump_start"`.
Or you can use integer `#define LOGDUMP_SAVE_START 1` and change conditions `if (atoi(buffer) == LOGDUMP_SAVE_START)` in log_dump_set_option

Reply:
`write` uses char arrays, so I want to use string.
comparing log string takes much time, so I don't want to use long string.
And two times of `atoi` also take much time.

Reply:
Is it impossible to use as integer without string converting?

Reply:
Possible, but I think this is better, because no need to convert many times.